_docs/tasks/traffic-management/ingress.md

```diff
@@ -205,21 +198,85 @@ The following are the known limitations of Istio ingress:
    EOF
    ```
 
-   Set the INGRESS_HOST to point to the ip address and the
-   port number of the ingress service as shown earlier.
+   > Note: Because SNI is not yet supported, Envoy currently only allows a single TLS secret in the ingress.
+   > That means the secretName field in ingress resource is not used.
 
-   > Note: Envoy currently only allows a single TLS secret in the ingress
-   > since SNI is not yet supported. That means that the secret name field
-   > in ingress resource is not used, and the secret must be called
```

Comment:
i believe you need this part, the cert must be called istio-ingress-certs

Reply:
OK. I can add it back to make it more explicit.

Reply:
Added to Line 170. Thanks.